drivers/SmartThings/philips-hue/profiles/legacy-color.yml

```diff
@@ -6,10 +6,6 @@ components:
     version: 1
   - id: switchLevel
     version: 1
-    config:
```

Comment:
Since the custom presentation isn't visible in this changeset, I wanted to make a visibility comment here to call out that these embedded device configs are removed in this PR because they're rolled in to the custom presentation upon which this PR depends.